### PR TITLE
Properly tweaks EMP

### PR DESCRIPTION
**Pull request by Sparrowbird611**
## About The Pull Request

Decreases EMP damage to cybernetics from the new values, and increases from the old values. These are

External 10/5/2

Internal 12/6/3

This translates to 70/35/24 damage to a full FBP externally per EMP. Please don't make me calculate internal damage, just know that for medium and heavy it's going to be in the hundreds if you count every organ that needs to be repaired.

## Why It's Good For The Game

The previous EMP PR was done very poorly, and tried to justify it with erismed 3. It was also a poor band-aid fix to nerf FBPs, when the solution is to fix the issues that were plaguing humans. These EMPs will do reasonable damage internal and external damage, reducing their efficiency and making them easier to kill or disable, while still taking into account that the majority of EMP sources are nigh impossible to dodge.

## Changelog
:cl:
balance: Adjusts the EMP damage dealt to FBPs. They are weaker now, but still capable of causing severe damage to internal and external machinery.
/:cl:

### CODE REVIEW

**Comment by MLGTASTICa** [comment]
The actual damage translates to 30 / 15 / 6 , because FBPs do not keep track of limb health. EMP's are a rare weapon with few uses and as such the damage shouldn't be this low,  it should be very punishing. Choosing to be a robot should yield robot consequences. Having it kept at high values is a much better fix than making EMP's instantly stun them.
It should also not take a full box of EMP's to kill FBP, with 6 nades , a synth can only get to 180 damage, its also worth mentioning that the damage dealt by EMP's is brute and not burn  , which makes it significantly easier to fix.
with your changes,  not even a full box of nades can brain-kill a FBP, a box can deal a total of 72 emp damage to organs.
This change only serves to ,  turn their only weakness into something that should not be cared about , if you going blind on a direct emp hit is too punishing , consider buffing robotic eye health from 50 to more.
Playing FBP yields many advantages , there is no reason to tweak down the damage values , EMP just began being used because NOW they actually deal damage to FBPs as they should , they weren't used before because the EMP Cannon  needs to land direct hits and only has 4 hits , and the other EMP's are not acquireable by IH without trade.
Sitting near to a cracker kills you , so should sitting next to a EMP nade as a FBP, getting hit by a rocket kills you , so should getting hit by an ion blast.

**Comment by Sparrowbird611** [comment]
> The actual damage translates to 30 / 15 / 6 , because FBPs do not keep track of limb health. EMP's are a rare weapon with few uses and as such the damage shouldn't be this low, it should be very punishing. Choosing to be a robot should yield robot consequences. Having it kept at high values is a much better fix than making EMP's instantly stun them.
> It should also not take a full box of EMP's to kill FBP, with 6 nades , a synth can only get to 180 damage, its also worth mentioning that the damage dealt by EMP's is brute and not burn , which makes it significantly easier to fix.
> with your changes, not even a full box of nades can brain-kill a FBP, a box can deal a total of 72 emp damage to organs.
> This change only serves to , turn their only weakness into something that should not be cared about , if you going blind on a direct emp hit is too punishing , consider buffing robotic eye health from 50 to more.
> Playing FBP yields many advantages , there is no reason to tweak down the damage values , EMP just began being used because NOW they actually deal damage to FBPs as they should , they weren't used before because the EMP Cannon needs to land direct hits and only has 4 hits , and the other EMP's are not acquireable by IH without trade.
> Sitting near to a cracker kills you , so should sitting next to a EMP nade as a FBP, getting hit by a rocket kills you , so should getting hit by an ion blast.

You neglect to include that the limbs still inflict pain, still begin to malfunction at 30 damage, and still effect speed by having their efficiency reduced (which EMPs do). You also neglect that there are PRs that will make it so humans do not have their limbs effect max HP either, so this point is moot.

EMPs are semi-rare. The guild can order effective grenades if they choose to, uranium, while I admit is semi-rare, is not incredibly difficult to obtain in levels that could be used for lethal numbers of EMPs, and traitors can purchase EMPs to their hearts content.

EMPs are primarily meant to be used against the environment, borgs, hive, and mechs. FBPs are a bonus. They have a wide range that disables multiple functions, including APCs, and in the case of grenades are undodgeable. Their damage reflects this.

You are correct, you cannot brain kill an FBP with a full box. He will however be completely blind, suffering from lung issues, heart problems, and other problems associated with low efficiency in organs, alongside completely destroyed nerves and muscles, before you completely expend your box.

This change serves to fix your mistake of making EMPs able to one shot FBPs, who did have some advantages, but did not justify having idiotic levels of organ and physical damage. Do not act like having pre-nerf laser cannon damage was justifiable. You will still deal significant damage to an FBP with these changes. EMPs were already fairly useful against FBPs, simply never used because shooting was more enjoyable. You also neglect that FBPs have other weaknesses, such as easier to weaken limbs, a lack of natural healing, and an inability to heal midfight with medkits and healing drugs. This is especially true given two of the three main advantages they have available already have PRs to negate them, so your band-aid fix is no longer necessary.

The halicon still deals 70 direct damage and decent levels of organ damage, with multiple shots this is completely crippling. I'm sorry you are incapable of things such as reloading and need a one shot weapon on par with pre-nerf laser cannon.

Then trade for EMP grenades if you want them so badly. IH could use some more trading.

A cracker and an EMP grenade are not the same thing. A halicon and an RPG are not the same thing. It would be idiotic to compare these weapons to each other. Use the cracker or RPG against an FBP, like you said, it kills them.

**Comment by MLGTASTICa** [comment]
> > The actual damage translates to 30 / 15 / 6 , because FBPs do not keep track of limb health. EMP's are a rare weapon with few uses and as such the damage shouldn't be this low, it should be very punishing. Choosing to be a robot should yield robot consequences. Having it kept at high values is a much better fix than making EMP's instantly stun them.
> > It should also not take a full box of EMP's to kill FBP, with 6 nades , a synth can only get to 180 damage, its also worth mentioning that the damage dealt by EMP's is brute and not burn , which makes it significantly easier to fix.
> > with your changes, not even a full box of nades can brain-kill a FBP, a box can deal a total of 72 emp damage to organs.
> > This change only serves to , turn their only weakness into something that should not be cared about , if you going blind on a direct emp hit is too punishing , consider buffing robotic eye health from 50 to more.
> > Playing FBP yields many advantages , there is no reason to tweak down the damage values , EMP just began being used because NOW they actually deal damage to FBPs as they should , they weren't used before because the EMP Cannon needs to land direct hits and only has 4 hits , and the other EMP's are not acquireable by IH without trade.
> > Sitting near to a cracker kills you , so should sitting next to a EMP nade as a FBP, getting hit by a rocket kills you , so should getting hit by an ion blast.
> 
> You neglect to include that the limbs still inflict pain, still begin to malfunction at 30 damage, and still effect speed by having their efficiency reduced (which EMPs do). All these things contribute to putting you into paincrit and disabling your combat efficiency. You also neglect that there are PRs that will make it so humans do not have their limbs effect max HP either, so this point is moot.
> 
> Do not try and compare FBPs to borgs. Borgs have no ability to feel pain and only stop moving when they are dead. FBPs are people who feel pain and are effected by most things that effect normal humans.
> 
> EMPs are semi-rare. The guild can order effective grenades if they choose to, uranium, while I admit is semi-rare, is not incredibly difficult to obtain in levels that could be used for lethal numbers of EMPs, and traitors can purchase EMPs to their hearts content.
> 
> EMPs are primarily meant to be used against the environment, borgs, hive, and mechs. FBPs are a bonus. They have a wide range that disables multiple functions, including APCs, and in the case of grenades are undodgeable. Their damage reflects this.
> 
> You are correct, you cannot brain kill an FBP with a full box. He will however be completely blind, suffering from lung issues, heart problems, and other problems associated with low efficiency in organs, alongside completely destroyed nerves and muscles, not to mention literally in paincrit, before you completely expend your box.
> 
> This change serves to fix your mistake of making EMPs able to one shot FBPs, who did have some advantages, but did not justify having idiotic levels of organ and physical damage. Do not act like having pre-nerf laser cannon damage was justifiable. You will still deal significant damage to an FBP with these changes. EMPs were already fairly useful against FBPs, simply never used because shooting was more enjoyable. You also neglect that FBPs have other weaknesses, such as easier to weaken limbs, a lack of natural healing, and an inability to heal midfight with medkits and healing drugs. This is especially true given two of the three main advantages they have available already have PRs to negate them, so your band-aid fix is no longer necessary.
> 
> The halicon still deals 70 direct damage and decent levels of organ damage, with multiple shots this is completely crippling. I'm sorry you are incapable of things such as landing multiple hits and reloading, and need a one shot weapon on par with pre-nerf laser cannon to compensate for that.
> 
> Then trade for EMP grenades if you want them so badly. IH could use some more trading.
> 
> A cracker and an EMP grenade are not the same thing. A halicon and an RPG are not the same thing. It would be idiotic to compare these weapons to each other, just as it would be idiotic to say a photon grenade or a fragmentation grenade are the same as a cracker. Use the cracker or RPG against an FBP, like you said, it kills them.

Destroyed nerves do not inflict pain,or damaged ones.

FBP's are still robotic in nature.

EMP's are meant to be used against mechanical threats

Organ efficiency is not a important factor when everyone is back to not using EMP's, and definetly won't dictate anything in combat when most of the time , nades will only deal 6.

EMP's were never used because they were barely effective and worth their usage when a gun could output more damage.

The halicon ,as specified earlier , only actualy deals 30 , and barely any organ damage , as you need to use most of its shot to actually cripple someone

Point is that they are rare

A cracker and EMP should be the same thing for a FBP , so should be an Halicon, they are robotic in nature , so should be their biggest weakness.

**Comment by SaltyOldBull** [comment]
people tried to negotiate damage with you when you made your PR, you didn't exactly budge. I hope you are not expecting people to not reciprocate this

**Comment by Sparrowbird611** [comment]
> > > The actual damage translates to 30 / 15 / 6 , because FBPs do not keep track of limb health. EMP's are a rare weapon with few uses and as such the damage shouldn't be this low, it should be very punishing. Choosing to be a robot should yield robot consequences. Having it kept at high values is a much better fix than making EMP's instantly stun them.
> > > It should also not take a full box of EMP's to kill FBP, with 6 nades , a synth can only get to 180 damage, its also worth mentioning that the damage dealt by EMP's is brute and not burn , which makes it significantly easier to fix.
> > > with your changes, not even a full box of nades can brain-kill a FBP, a box can deal a total of 72 emp damage to organs.
> > > This change only serves to , turn their only weakness into something that should not be cared about , if you going blind on a direct emp hit is too punishing , consider buffing robotic eye health from 50 to more.
> > > Playing FBP yields many advantages , there is no reason to tweak down the damage values , EMP just began being used because NOW they actually deal damage to FBPs as they should , they weren't used before because the EMP Cannon needs to land direct hits and only has 4 hits , and the other EMP's are not acquireable by IH without trade.
> > > Sitting near to a cracker kills you , so should sitting next to a EMP nade as a FBP, getting hit by a rocket kills you , so should getting hit by an ion blast.
> > 
> > 
> > You neglect to include that the limbs still inflict pain, still begin to malfunction at 30 damage, and still effect speed by having their efficiency reduced (which EMPs do). All these things contribute to putting you into paincrit and disabling your combat efficiency. You also neglect that there are PRs that will make it so humans do not have their limbs effect max HP either, so this point is moot.
> > Do not try and compare FBPs to borgs. Borgs have no ability to feel pain and only stop moving when they are dead. FBPs are people who feel pain and are effected by most things that effect normal humans.
> > EMPs are semi-rare. The guild can order effective grenades if they choose to, uranium, while I admit is semi-rare, is not incredibly difficult to obtain in levels that could be used for lethal numbers of EMPs, and traitors can purchase EMPs to their hearts content.
> > EMPs are primarily meant to be used against the environment, borgs, hive, and mechs. FBPs are a bonus. They have a wide range that disables multiple functions, including APCs, and in the case of grenades are undodgeable. Their damage reflects this.
> > You are correct, you cannot brain kill an FBP with a full box. He will however be completely blind, suffering from lung issues, heart problems, and other problems associated with low efficiency in organs, alongside completely destroyed nerves and muscles, not to mention literally in paincrit, before you completely expend your box.
> > This change serves to fix your mistake of making EMPs able to one shot FBPs, who did have some advantages, but did not justify having idiotic levels of organ and physical damage. Do not act like having pre-nerf laser cannon damage was justifiable. You will still deal significant damage to an FBP with these changes. EMPs were already fairly useful against FBPs, simply never used because shooting was more enjoyable. You also neglect that FBPs have other weaknesses, such as easier to weaken limbs, a lack of natural healing, and an inability to heal midfight with medkits and healing drugs. This is especially true given two of the three main advantages they have available already have PRs to negate them, so your band-aid fix is no longer necessary.
> > The halicon still deals 70 direct damage and decent levels of organ damage, with multiple shots this is completely crippling. I'm sorry you are incapable of things such as landing multiple hits and reloading, and need a one shot weapon on par with pre-nerf laser cannon to compensate for that.
> > Then trade for EMP grenades if you want them so badly. IH could use some more trading.
> > A cracker and an EMP grenade are not the same thing. A halicon and an RPG are not the same thing. It would be idiotic to compare these weapons to each other, just as it would be idiotic to say a photon grenade or a fragmentation grenade are the same as a cracker. Use the cracker or RPG against an FBP, like you said, it kills them.
> 
> Destroyed nerves do not inflict pain,or damaged ones.
> 
> FBP's are still robotic in nature.
> 
> EMP's are meant to be used against mechanical threats
> 
> Organ efficiency is not a important factor when everyone is back to not using EMP's, and definetly won't dictate anything in combat when most of the time , nades will only deal 6.
> 
> EMP's were never used because they were barely effective and worth their usage when a gun could output more damage.
> 
> The halicon ,as specified earlier , only actualy deals 30 , and barely any organ damage , as you need to use most of its shot to actually cripple someone
> 
> Point is that they are rare
> 
> A cracker and EMP should be the same thing for a FBP , so should be an Halicon, they are robotic in nature , so should be their biggest weakness.
> 
> I do agree that that instant eye blind is bullshit , it should be 2 shot and i really suggest decreasing internal organ damage to 20, but the old values are way too low.

Use EMPs against mechanical threats. They will still work, they just won't be instant wins.

Destroyed nerves are reduced efficiency, causing speed loss and weapon dropping at high levels, both of which are significant disadvantages.

A halicon deals seventy damage. All of this damage contributes to paincrit. You don't get to say otherwise when the numbers say something else entirely.

EMPs were never used because FBPs are first off, rather rare opponents, and second off, less enjoyable to use than guns in most cases. The times I had seen EMPs used in the past, they were quite effective against mechanical foes.

They are not rare. A cracker is rare, being only available in a traitor PDA or the rare NT disk, an EMP is semi-common, being both in the IH armory roundstart, easily orderable by the guild, somewhat easy to make in chemistry, cheap in the traitor PDA, and overall effective in all these circumstances.

If you want to tell me that a cracker should be on par with an EMP, good for you. I don't agree and I don't care. EMPs have superior range over crackers and disable entire rooms, they deal decent levels of damage for what they are.

I have done my tests on FBPs. 1 medium and 1 heavy will produce half blindness, 1 med and 2 heavy will produce full blindness. I don't recall if two heavy shots produce full blindness or not, but I know they at very least come close, and three shots from a halicon are all but guaranteed to paincrit an FBP by dealing 210 damage externally, make every limb need surgery to be repaired, cause the affecteed FBP to begin to suffocate and have oxy-loss in the brain, and cause every limb to malfunctions. If a weapon that can deal this much damage in three shots isn't acceptable to you, then frankly I can't say I care about pleasing you.

As it stands, humans are already likely to receive a buff soon. The advantages of FBPs will be significantly lessened due to this, so I feel no obligation to increase the damage that EMPs deal higher than the levels presented in this PR. This is, quite frankly, the most balanced option.

**Comment by MLGTASTICa** [comment]
> > > > The actual damage translates to 30 / 15 / 6 , because FBPs do not keep track of limb health. EMP's are a rare weapon with few uses and as such the damage shouldn't be this low, it should be very punishing. Choosing to be a robot should yield robot consequences. Having it kept at high values is a much better fix than making EMP's instantly stun them.
> > > > It should also not take a full box of EMP's to kill FBP, with 6 nades , a synth can only get to 180 damage, its also worth mentioning that the damage dealt by EMP's is brute and not burn , which makes it significantly easier to fix.
> > > > with your changes, not even a full box of nades can brain-kill a FBP, a box can deal a total of 72 emp damage to organs.
> > > > This change only serves to , turn their only weakness into something that should not be cared about , if you going blind on a direct emp hit is too punishing , consider buffing robotic eye health from 50 to more.
> > > > Playing FBP yields many advantages , there is no reason to tweak down the damage values , EMP just began being used because NOW they actually deal damage to FBPs as they should , they weren't used before because the EMP Cannon needs to land direct hits and only has 4 hits , and the other EMP's are not acquireable by IH without trade.
> > > > Sitting near to a cracker kills you , so should sitting next to a EMP nade as a FBP, getting hit by a rocket kills you , so should getting hit by an ion blast.
> > > 
> > > 
> > > You neglect to include that the limbs still inflict pain, still begin to malfunction at 30 damage, and still effect speed by having their efficiency reduced (which EMPs do). All these things contribute to putting you into paincrit and disabling your combat efficiency. You also neglect that there are PRs that will make it so humans do not have their limbs effect max HP either, so this point is moot.
> > > Do not try and compare FBPs to borgs. Borgs have no ability to feel pain and only stop moving when they are dead. FBPs are people who feel pain and are effected by most things that effect normal humans.
> > > EMPs are semi-rare. The guild can order effective grenades if they choose to, uranium, while I admit is semi-rare, is not incredibly difficult to obtain in levels that could be used for lethal numbers of EMPs, and traitors can purchase EMPs to their hearts content.
> > > EMPs are primarily meant to be used against the environment, borgs, hive, and mechs. FBPs are a bonus. They have a wide range that disables multiple functions, including APCs, and in the case of grenades are undodgeable. Their damage reflects this.
> > > You are correct, you cannot brain kill an FBP with a full box. He will however be completely blind, suffering from lung issues, heart problems, and other problems associated with low efficiency in organs, alongside completely destroyed nerves and muscles, not to mention literally in paincrit, before you completely expend your box.
> > > This change serves to fix your mistake of making EMPs able to one shot FBPs, who did have some advantages, but did not justify having idiotic levels of organ and physical damage. Do not act like having pre-nerf laser cannon damage was justifiable. You will still deal significant damage to an FBP with these changes. EMPs were already fairly useful against FBPs, simply never used because shooting was more enjoyable. You also neglect that FBPs have other weaknesses, such as easier to weaken limbs, a lack of natural healing, and an inability to heal midfight with medkits and healing drugs. This is especially true given two of the three main advantages they have available already have PRs to negate them, so your band-aid fix is no longer necessary.
> > > The halicon still deals 70 direct damage and decent levels of organ damage, with multiple shots this is completely crippling. I'm sorry you are incapable of things such as landing multiple hits and reloading, and need a one shot weapon on par with pre-nerf laser cannon to compensate for that.
> > > Then trade for EMP grenades if you want them so badly. IH could use some more trading.
> > > A cracker and an EMP grenade are not the same thing. A halicon and an RPG are not the same thing. It would be idiotic to compare these weapons to each other, just as it would be idiotic to say a photon grenade or a fragmentation grenade are the same as a cracker. Use the cracker or RPG against an FBP, like you said, it kills them.
> > 
> > 
> > Destroyed nerves do not inflict pain,or damaged ones.
> > FBP's are still robotic in nature.
> > EMP's are meant to be used against mechanical threats
> > Organ efficiency is not a important factor when everyone is back to not using EMP's, and definetly won't dictate anything in combat when most of the time , nades will only deal 6.
> > EMP's were never used because they were barely effective and worth their usage when a gun could output more damage.
> > The halicon ,as specified earlier , only actualy deals 30 , and barely any organ damage , as you need to use most of its shot to actually cripple someone
> > Point is that they are rare
> > A cracker and EMP should be the same thing for a FBP , so should be an Halicon, they are robotic in nature , so should be their biggest weakness.
> > I do agree that that instant eye blind is bullshit , it should be 2 shot and i really suggest decreasing internal organ damage to 20, but the old values are way too low.
> 
> Use EMPs against mechanical threats. They will still work, they just won't be instant wins.
> 
> Destroyed nerves are reduced efficiency, causing speed loss and weapon dropping at high levels, both of which are significant disadvantages.
> 
> A halicon deals seventy damage. All of this damage contributes to paincrit. You don't get to say otherwise when the numbers say something else entirely. All that matters is that the target is paincrit for an execution.
> 
> EMPs were never used because FBPs are first off, rather rare opponents, and second off, less enjoyable to use than guns in most cases, not to mention not being high on most people's acquisition list. The times I had seen EMPs used in the past, they were quite effective against mechanical foes.
> 
> They are not rare. A cracker is rare, being only available in a traitor PDA or the rare NT disk, an EMP is semi-common, being both in the IH armory roundstart, easily orderable by the guild, somewhat easy to make in chemistry, cheap in the traitor PDA, and overall effective in all these circumstances.
> 
> If you want to tell me that a cracker should be on par with an EMP, good for you. I don't agree and I don't care. EMPs have superior range over crackers and disable entire rooms, they deal decent levels of damage for what they are.
> 
> I have done my tests on FBPs. 1 medium and 1 heavy will produce half blindness, 1 med and 2 heavy will produce full blindness. I don't recall if two heavy shots produce full blindness or not, but I know they at very least come close, and three shots from a halicon are all but guaranteed to paincrit an FBP by dealing 210 damage externally, make every limb need surgery to be repaired, cause the affecteed FBP to begin to suffocate and have oxy-loss in the brain, and cause every limb to malfunctions. If a weapon that can deal this much damage in three shots isn't acceptable to you, then frankly I can't say I care about pleasing you.
> 
> As it stands, humans are already likely to receive a buff soon, with several PRs already lined up. The advantages of FBPs will be significantly lessened due to this, so I feel no obligation to increase the damage that EMPs deal higher than the levels presented in this PR. This is, quite frankly, the most balanced option.

Destroyed nerves only cause slight slowdown on the legs , they don't cause you to drop items from hands.

Only 30 brute actually contributes to the paincrit , due to how limbs works for FBPs

Ill repeat again , EMP's were not used because they have to be ordered , IH does not get any EMP grenades , only an EMP cannon that has to land direct hits and has only 4 shots with the medium cells, assault rifles were preffered because they could far out-do emp's just because of the ability to fire non-stop and more damage per magazine.

EMP's are rare , they have to be ordered , they rarely spawn in maint and IH only start with 2 EMP guns.

Yes , a EMP should be on-par with a cracker for FBP's , its how EMP's are to current cyborgs , you choose to be a robot you get robot consequences.

These are just my arguments , it was up to firefox to merge the damage buff , and hopefully , he does not merge this full revert , as it will make once again, EMP's a  far too uneffective against FBP's.

**Comment by Sparrowbird611** [comment]
> > > > > The actual damage translates to 30 / 15 / 6 , because FBPs do not keep track of limb health. EMP's are a rare weapon with few uses and as such the damage shouldn't be this low, it should be very punishing. Choosing to be a robot should yield robot consequences. Having it kept at high values is a much better fix than making EMP's instantly stun them.
> > > > > It should also not take a full box of EMP's to kill FBP, with 6 nades , a synth can only get to 180 damage, its also worth mentioning that the damage dealt by EMP's is brute and not burn , which makes it significantly easier to fix.
> > > > > with your changes, not even a full box of nades can brain-kill a FBP, a box can deal a total of 72 emp damage to organs.
> > > > > This change only serves to , turn their only weakness into something that should not be cared about , if you going blind on a direct emp hit is too punishing , consider buffing robotic eye health from 50 to more.
> > > > > Playing FBP yields many advantages , there is no reason to tweak down the damage values , EMP just began being used because NOW they actually deal damage to FBPs as they should , they weren't used before because the EMP Cannon needs to land direct hits and only has 4 hits , and the other EMP's are not acquireable by IH without trade.
> > > > > Sitting near to a cracker kills you , so should sitting next to a EMP nade as a FBP, getting hit by a rocket kills you , so should getting hit by an ion blast.
> > > > 
> > > > 
> > > > You neglect to include that the limbs still inflict pain, still begin to malfunction at 30 damage, and still effect speed by having their efficiency reduced (which EMPs do). All these things contribute to putting you into paincrit and disabling your combat efficiency. You also neglect that there are PRs that will make it so humans do not have their limbs effect max HP either, so this point is moot.
> > > > Do not try and compare FBPs to borgs. Borgs have no ability to feel pain and only stop moving when they are dead. FBPs are people who feel pain and are effected by most things that effect normal humans.
> > > > EMPs are semi-rare. The guild can order effective grenades if they choose to, uranium, while I admit is semi-rare, is not incredibly difficult to obtain in levels that could be used for lethal numbers of EMPs, and traitors can purchase EMPs to their hearts content.
> > > > EMPs are primarily meant to be used against the environment, borgs, hive, and mechs. FBPs are a bonus. They have a wide range that disables multiple functions, including APCs, and in the case of grenades are undodgeable. Their damage reflects this.
> > > > You are correct, you cannot brain kill an FBP with a full box. He will however be completely blind, suffering from lung issues, heart problems, and other problems associated with low efficiency in organs, alongside completely destroyed nerves and muscles, not to mention literally in paincrit, before you completely expend your box.
> > > > This change serves to fix your mistake of making EMPs able to one shot FBPs, who did have some advantages, but did not justify having idiotic levels of organ and physical damage. Do not act like having pre-nerf laser cannon damage was justifiable. You will still deal significant damage to an FBP with these changes. EMPs were already fairly useful against FBPs, simply never used because shooting was more enjoyable. You also neglect that FBPs have other weaknesses, such as easier to weaken limbs, a lack of natural healing, and an inability to heal midfight with medkits and healing drugs. This is especially true given two of the three main advantages they have available already have PRs to negate them, so your band-aid fix is no longer necessary.
> > > > The halicon still deals 70 direct damage and decent levels of organ damage, with multiple shots this is completely crippling. I'm sorry you are incapable of things such as landing multiple hits and reloading, and need a one shot weapon on par with pre-nerf laser cannon to compensate for that.
> > > > Then trade for EMP grenades if you want them so badly. IH could use some more trading.
> > > > A cracker and an EMP grenade are not the same thing. A halicon and an RPG are not the same thing. It would be idiotic to compare these weapons to each other, just as it would be idiotic to say a photon grenade or a fragmentation grenade are the same as a cracker. Use the cracker or RPG against an FBP, like you said, it kills them.
> > > 
> > > 
> > > Destroyed nerves do not inflict pain,or damaged ones.
> > > FBP's are still robotic in nature.
> > > EMP's are meant to be used against mechanical threats
> > > Organ efficiency is not a important factor when everyone is back to not using EMP's, and definetly won't dictate anything in combat when most of the time , nades will only deal 6.
> > > EMP's were never used because they were barely effective and worth their usage when a gun could output more damage.
> > > The halicon ,as specified earlier , only actualy deals 30 , and barely any organ damage , as you need to use most of its shot to actually cripple someone
> > > Point is that they are rare
> > > A cracker and EMP should be the same thing for a FBP , so should be an Halicon, they are robotic in nature , so should be their biggest weakness.
> > > I do agree that that instant eye blind is bullshit , it should be 2 shot and i really suggest decreasing internal organ damage to 20, but the old values are way too low.
> > 
> > 
> > Use EMPs against mechanical threats. They will still work, they just won't be instant wins.
> > Destroyed nerves are reduced efficiency, causing speed loss and weapon dropping at high levels, both of which are significant disadvantages.
> > A halicon deals seventy damage. All of this damage contributes to paincrit. You don't get to say otherwise when the numbers say something else entirely. All that matters is that the target is paincrit for an execution.
> > EMPs were never used because FBPs are first off, rather rare opponents, and second off, less enjoyable to use than guns in most cases, not to mention not being high on most people's acquisition list. The times I had seen EMPs used in the past, they were quite effective against mechanical foes.
> > They are not rare. A cracker is rare, being only available in a traitor PDA or the rare NT disk, an EMP is semi-common, being both in the IH armory roundstart, easily orderable by the guild, somewhat easy to make in chemistry, cheap in the traitor PDA, and overall effective in all these circumstances.
> > If you want to tell me that a cracker should be on par with an EMP, good for you. I don't agree and I don't care. EMPs have superior range over crackers and disable entire rooms, they deal decent levels of damage for what they are.
> > I have done my tests on FBPs. 1 medium and 1 heavy will produce half blindness, 1 med and 2 heavy will produce full blindness. I don't recall if two heavy shots produce full blindness or not, but I know they at very least come close, and three shots from a halicon are all but guaranteed to paincrit an FBP by dealing 210 damage externally, make every limb need surgery to be repaired, cause the affecteed FBP to begin to suffocate and have oxy-loss in the brain, and cause every limb to malfunctions. If a weapon that can deal this much damage in three shots isn't acceptable to you, then frankly I can't say I care about pleasing you.
> > As it stands, humans are already likely to receive a buff soon, with several PRs already lined up. The advantages of FBPs will be significantly lessened due to this, so I feel no obligation to increase the damage that EMPs deal higher than the levels presented in this PR. This is, quite frankly, the most balanced option.
> 
> Destroyed nerves only cause slight slowdown on the legs , they don't cause you to drop items from hands.
> 
> Only 30 brute actually contributes to the paincrit , due to how limbs works for FBPs
> 
> Ill repeat again , EMP's were not used because they have to be ordered , IH does not get any EMP grenades , only an EMP cannon that has to land direct hits and has only 4 shots with the medium cells, assault rifles were preffered because they could far out-do emp's just because of the ability to fire non-stop and more damage per magazine.
> 
> EMP's are rare , they have to be ordered , they rarely spawn in maint and IH only start with 2 EMP guns.
> 
> Yes , a EMP should be on-par with a cracker for FBP's , its how EMP's are to current cyborgs , you choose to be a robot you get robot consequences.
> 
> These are just my arguments , it was up to firefox to merge the damage buff , and hopefully , he does not merge this full revert , as it will make once again, EMP's a far too uneffective against FBP's.

You're blatantly wrong on the limbs. I've been paincrit by having my robotic legs damaged too much, and nothing else.

EMPs are semi-common, I'm not budging on this. There's quite a few ways to acquire them.

I'm not going to say your opinion is wrong, but I'd like to.

To be frank, the previous PR shouldn't have even been merged. This is not a full revert, this is a tested, tried, and balanced EMP PR, and your stubbornness and lack of balancing competency should not be allowed to get in the way of it.

**Comment by Sparrowbird611** [comment]
Also HOLY FUCK, can we mutually agree that if either of us have anything else to say, we don't quote it?

**Comment by Sparrowbird611** [comment]
Also I just wanted to mention something about your cracker/EMP comparison. after some testing, a cracker does deals similar damage to a person in average armor (10% bomb resist) as an EMP in this PR does at similar ranges, minus the organ rape. Speaking of armor, a cracker also respects armor and has a shorter range than an EMP does (about 4-5 tiles compared to like 10-15 depending on the EMP grenade). So if you really want to compare a cracker to an EMP, this PR comes closer to it than the previous, as a cracker will deal on average 80 damage per grenade, with minimal if any organ damage, which will take 3 grenades to reach 240 damage. An EMP in this PR will deal 210 damage at the same range, on top of a good deal of organ rape, and you will still be rewarded even if your opponent escapes the blast radius, so I'd frankly consider the EMP weaponry stronger damage wise in this scenario.

If your main argument against this is that limbs don't count, then you may want to start arguing with jack on the PR that makes human limbs not count to total HP, because otherwise that argument will be completely moot by friday. Either way, the limbs do still contribute to pain, so you'll probably crit the people before you ever reach three grenades if you let the pain set in first (because pain funnily enough takes like 5 seconds to catch up in Eris).

**Comment by TheShown911** [comment]
Too long, didn't read.

**Comment by Sparrowbird611** [comment]
> Too long, didn't read.

Based. If you want the TL:DR, nerfs EMPs. They'll do slightly lower damage than crackers externally, and far more internal damage, with far more range as well.

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**
